### PR TITLE
Update Getting Started guide

### PR DESCRIPTION
**Pull request by space-pope**
It still needs an update or two (pending the completion of one of the APIs), but I thought maybe it'd help to have this for people to look over.

### CODE REVIEW

File: content/docs/Getting-Started/index.md [hunk 1]
```diff
@@ -5,17 +5,125 @@ description: Getting started with the Spokestack API
...
+One caveat before we start, though: _This is not a collection of best practices_. We're going to be trading thoughtful organization for convenience here, so when we say something like "put this in your view controller", just know that you might not want to leave it there long-term. OK, now that that's out of the way, let's jump right in.
 
-React is a great way to organize speech components. W3C’s Web Speech API browser support is disappointing. If you’re implementing React components that use the Web Speech API, we recommend only supporting the Chrome browser if possible. Otherwise, build your own custom React components that utilize a 3rd-party ASR service instead of Web Speech API’s ASR while using Web Speech API TTS.
+## Installation
 
-## Protocol
+The rest of the guide will assume you already have the Spokestack framework installed in your project. See [the README](https://github.com/spokestack/spokestack-ios/blob/master/README.md) for more information, but if you're using [CocoaPods](https://cocoapods.org/), it's as easy as adding
+```
+pod 'Spokestack-iOS'
+```
```
Comment (timmywil) [inline content/docs/Getting-Started/index.md:17]: Specify bash as language.

File: content/docs/Getting-Started/index.md [hunk 2]
```diff
@@ -5,17 +5,125 @@ description: Getting started with the Spokestack API
...
+to your `Podfile` and running
+```
+pod install
+```
```
Comment (timmywil) [inline content/docs/Getting-Started/index.md:21]: Same here

File: content/docs/Getting-Started/index.md [hunk 3]
```diff
@@ -5,17 +5,125 @@ description: Getting started with the Spokestack API
...
+
+For the first one, head over to `Info.plist` in your project and add a couple keys. Here are the raw values you'll need (the keys, at least; feel free to substitute your own values):
+
+
+```
+	<key>NSMicrophoneUsageDescription</key>
+	<string>The microphone is used to receive user commands via voice.</string>
+	<key>NSSpeechRecognitionUsageDescription</key>
+	<string>Speech recognition is used to translate user voice input into text for further processing.</string>
+```
```
Comment (timmywil) [inline content/docs/Getting-Started/index.md:41]: Specify xml as language

**Review by timmywil — CHANGES_REQUESTED** [review]
Looks good. Always specify a language so code gets colors.

File: content/docs/Getting-Started/index.md (hunk 3 above)
Comment (timmywil) [reply content/docs/Getting-Started/index.md:41]: I'd also remove the extra tabs here.

Comment (space-pope) [reply content/docs/Getting-Started/index.md:41]: Good call—and gah; didn't even realize those were actual tabs. Thanks, Xcode.

**Comment by space-pope** [comment]
Thanks for the syntax check. I sort of didn't think the `bash` would do anything for the CocoaPods commands, but it's a good habit either way.

**Pull request merged**